R/qc_missed_cleavages.R

```diff
@@ -21,7 +21,13 @@
 #'
 #' @examples
 #' \dontrun{
-#' qc_missed_cleavages(data, sample, grouping, missed_cleavages, intensity, method = "intensity", plot = TRUE)
+#' qc_missed_cleavages(data,
```

Comment:
would do the same as above with putting the data in new line and adding spectronaut column names. since this is the example code of how to use the function.